### PR TITLE
getCoords with fixedRowsBottom after scroll return appropriate coords

### PR DESCRIPTION
**Pull request by pnowak**
### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #6084

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

File: src/3rdparty/walkontable/src/table.js [hunk 1]
```diff
@@ -787,7 +786,7 @@ class Table {
   }
 
   isRowAfterRenderedRows(row) {
-    return this.rowFilter && (row > this.getLastRenderedRow());
+    return this.rowFilter && (this.rowFilter.sourceToRendered(row) > this.getLastRenderedRow());
```
Comment (wojciechczerniak) [inline src/3rdparty/walkontable/src/table.js]: This was recently changed by @warpech PR. Please communicate before changing it back and forth @jansiegel @warpech @pnowak  

https://github.com/handsontable/handsontable/pull/6079/files#diff-0a25aaaa72d09482c0875ad092179097L567

Comment (jansiegel) [reply src/3rdparty/walkontable/src/table.js]: Yeah, my change didn't work properly, it's reverted in the next commit.

File: src/3rdparty/walkontable/test/spec/table.spec.js [hunk 2]
```diff
@@ -120,6 +120,21 @@ describe('WalkontableTable', () => {
...
+    const wt = walkontable({
+      data: getData,
+      fixedRowsBottom: 2,
+      totalRows: getTotalRows,
+      totalColumns: getTotalColumns
+    });
+    wt.draw();
+
+    const bottomTable = wt.wtOverlays.bottomOverlay.clone.wtTable;
+    expect(bottomTable.getCell(new Walkontable.CellCoords(18, 0)) instanceof HTMLElement).toBe(true);
```
Comment (swistach) [inline src/3rdparty/walkontable/test/spec/table.spec.js]:
```suggestion
    expect(bottomTable.getCell(new Walkontable.CellCoords(18, 0)) instanceof HTMLTableCellElement).toBe(true);
```

File: src/3rdparty/walkontable/test/spec/table.spec.js [hunk 3]
```diff
@@ -120,6 +120,21 @@ describe('WalkontableTable', () => {
...
+    expect(bottomTable.getCell(new Walkontable.CellCoords(19, 0)) instanceof HTMLElement).toBe(true);
```
Comment (swistach) [inline src/3rdparty/walkontable/test/spec/table.spec.js]:
```suggestion
    expect(bottomTable.getCell(new Walkontable.CellCoords(19, 0)) instanceof HTMLTableCellElement).toBe(true);
```

**Pull request merged**